README.md

```diff
@@ -191,6 +192,24 @@ There are various tools to look XUIDs up online and they are also printed to the
 -e ALLOW_LIST_USERS="player1:1234567890,player2:0987654321"
 ```
 
+## Variables
+
+Custom server variables are passed in just like the allowlist or as a full JSON string.
+
+Server variables are parsed into their most likely type (number-like turn into numbers, all other inputs are treated as string) using jq's `fromjson` command. In the example below, `var1` is a string, `var2` is a number, and `var3` is a string. 
+
+For greater control on types, users can provide a full string JSON representation that is used as-is.
+
+All variables are written to the config/default/variables.json. There is no support for Module-specific variable handling at this time.
+
+```shell
+# passing in simple expressions
+-e VARIABLES="var1:customStringValue,var2:1234,var3:true"
```

Comment:
It seems like in non-json form it would be more intuitive to use equal signs such as

var1=value,var2=value

Reply:
Do we want to diverge from the format for the allowlist variable? I only kept using colons to follow the previous input formatting. not an issue to change if we're okay with different formats for different env var inputs.

Reply:
I think that's ok since your scenario is a definitive name=value whereas the allowlist is a pairing of values due to the XUID weirdness.